src/api/init.ts

```diff
@@ -149,6 +152,10 @@ export async function startApiServer(opts: {
     res.redirect(`/extended/v1/status`);
   });
 
+  if (!isProdEnv) {
```

Comment:
We should serve this on prod too... thoughts @CharlieC3 @saralab ?

Reply:
I don't see why it shouldn't always be enabled. Is there any reason this enablement should be controlled by an ENV var?